simulator/source/Combat_simulator.cpp

```diff
@@ -1249,7 +1284,12 @@ void Combat_simulator::simulate(const Character& character, const std::function<
                 {
```

Comment:
pre-existing: I'd rather stop if `sunder_armor_stacks_ >= sunder_armor_globals`, so misses and dodges are honored (instead of `sunder_armors_stacks_ >= 5`)

Reply:
Both options seem reasonable to me, it really depends on what the user's intent is when they choose the value for sunder armor globals. When most warrior say "sunder first global" they probably aren't going to recast sunder if that misses. However, a warrior who has the job like stacking 5 sunders on a mob where a druid might be tanking and there are no other warriors in the raid would want the behavior you're describing. In TBC where dodge is in the ~1-2% range for most people at current gear levels, in my opinion this might not be worth worrying about.

Reply:
True, but I guess the original (from vanilla-classic) intent of this option was to cover the DPS detriment of the additional "get up sunders asap" job for some of the 15 warriors ;) With one or two warriors left, I'd rather correctly support the "stack 5 sunders" semantic.

Reply:
It appears that this is settled then? Let me know if there's some smarter behavior that you think needs to be added here.